diff -dur lmms-1.2.0-rc7.orig/cmake/modules/BashCompletion.cmake lmms-1.2.0-rc7/cmake/modules/BashCompletion.cmake
--- lmms-1.2.0-rc7.orig/cmake/modules/BashCompletion.cmake	2018-09-25 09:47:51.000000000 +0200
+++ lmms-1.2.0-rc7/cmake/modules/BashCompletion.cmake	2018-11-20 21:01:39.306149303 +0100
@@ -78,9 +78,9 @@
   BASHCOMP_PKG_PATH=\"${BASHCOMP_PKG_PATH}\"\n\
 fi\n\
 echo -e \"\\nInstalling bash completion...\\n\"\n\
-mkdir -p \"\$BASHCOMP_PKG_PATH\"\n\
-cp \"${CMAKE_CURRENT_SOURCE_DIR}/${SCRIPT_NAME}\" \"\$BASHCOMP_PKG_PATH\"\n\
-chmod a+r \"\$BASHCOMP_PKG_PATH/${SCRIPT_NAME}\"\n\
+mkdir -p \"\$DESTDIR\$BASHCOMP_PKG_PATH\"\n\
+cp \"${CMAKE_CURRENT_SOURCE_DIR}/${SCRIPT_NAME}\" \"\$DESTDIR\$BASHCOMP_PKG_PATH\"\n\
+chmod a+r \"\$DESTDIR\$BASHCOMP_PKG_PATH/${SCRIPT_NAME}\"\n\
 echo -e \"Bash completion for ${SCRIPT_NAME} has been installed to \$BASHCOMP_PKG_PATH/${SCRIPT_NAME}\"\n\
 ")
 		INSTALL(CODE "EXECUTE_PROCESS(COMMAND chmod u+x \"install_${SCRIPT_NAME}_completion.sh\" WORKING_DIRECTORY ${CMAKE_CURRENT_BINARY_DIR} )")
