diff -uNr lmms-0.4.13.old/plugins/zynaddsubfx/LocalZynAddSubFx.cpp lmms-0.4.13/plugins/zynaddsubfx/LocalZynAddSubFx.cpp
--- lmms-0.4.13.old/plugins/zynaddsubfx/LocalZynAddSubFx.cpp	2012-02-02 23:27:02.000000000 +0100
+++ lmms-0.4.13/plugins/zynaddsubfx/LocalZynAddSubFx.cpp	2012-09-15 22:59:23.599016916 +0200
@@ -22,6 +22,8 @@
  *
  */
 
+#include <unistd.h>
+
 #include <lmmsconfig.h>
 
 #include "LocalZynAddSubFx.h"
